docs/guides/getting-started/setup/_fragments/_tauri-init.mdx

```diff
@@ -16,14 +83,11 @@ It will walk you through a series of questions:
 2. **What should the window title be?**  
    This will be the title of the default main window. You can use any title you want here.
 
-3. **Where are your web assets (HTML/CSS/JS) located relative to the "<current dir\>/src-tauri/tauri.conf.json" file that will be created?**  
-   This is the path that Tauri will load your frontend assets from when building for **production**.
-
-   <p>{props.webAssets}</p>
+3. **Where are your web assets (HTML/CSS/JS) located relative to the `<current dir>/src-tauri/tauri.conf.json` file that will be created?**  
+   This is the path that Tauri will load your frontend assets from when building for **production**. Use the value mentioned previously.
 
 4. **What is the URL of your dev server?**  
-   This can be either a URL or a file path that Tauri will load during **development**.
-   <span>{props.devServer}</span>
+   This can be either a URL or a file path that Tauri will load during **development**. Use the value mentioned previously.
```

Comment:
Is it possible to give the fragments an argument? I'd prefer to write the actual answer here on top of "Use the value mentioned previously.", i spent too much time searching where it was "mentioned previously" right now :D

Reply:
We had fragments but they messed up i18n a lot. But let me look at changing it where you only pass the actual VALUE and not a phrase since you won't need to i18n the value

Reply:
This should be updated now